.travis.yml

```diff
@@ -4,5 +4,10 @@ jdk:
 before_install:
   - "echo $JAVA_OPTS"
   - "export JAVA_OPTS=-Xmx512m"
+
+script:
+  - mvn javadoc:jar
+  - mvn javadoc:test-aggregate
```

Comment:
I see this in the travis build posted above:
https://travis-ci.org/fcrepo4-labs/fcrepo-audit#L89
"mvn install -DskipTests=true -Dmaven.javadoc.skip=true -B -V" 

That must be a default command?